x-pack/plugins/search_assistant/kibana.jsonc

```diff
@@ -16,6 +16,7 @@
     "requiredPlugins": [
       "actions",
       "licensing",
+      "ml",
```

Comment:
Maybe we should add a comment here explaining the nature of the dep (since it's no in code someone might think it is OK to remove it)